public/js/components/FormFields/FormFieldArrayWrapper.js

```diff
@@ -76,10 +101,24 @@ export default class FormFieldArrayWrapper extends React.Component {
       });
     });
 
+    const returnMoveBtns = () => {
```

Comment:
`renderMoveButtons` would be more consistent with other examples of these functions in the app